packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -351,6 +354,31 @@ class MediaQueryData {
   /// gesture behavior over the framework constants.
   final DeviceGestureSettings gestureSettings;
 
+  /// {@macro dart.ui.ViewConfiguration.displayFeatures}
+  ///
+  /// See also:
+  ///
+  ///  * [dart:ui.DisplayFeatureType], where you can see types of display features.
+  ///  * [dart:ui.DisplayFeatureState], where you can see possible states for
+  ///  folding features ([dart:ui.DisplayFeatureType.fold] and
+  ///  [dart:ui.DisplayFeatureType.hinge])
+  final List<ui.DisplayFeature> displayFeatures;
+
+
+  /// Area of the display that is obstructed specifically by the [DisplayFeatureType.hinge].
```

Comment:
nit: keep these to about 80chars, https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#prefer-a-maximum-line-length-of-80-characters